src/db/timesheets/Formatter.ts

```diff
@@ -0,0 +1,96 @@
+import * as timesheetSchemas from 'src/db/schemas/Timesheet'
+
+import * as constants from 'src/constants'
+import { v4 as uuidv4 } from 'uuid';
+
+import {UserTimesheets, WriteEntryToTable} from 'src/dynamodb'
+import { DBToFrontend } from './FrontendConversions';
+
+const moment = require('moment-timezone'); 
+
+
+export class Formatter {
+    /*
+        Processes the timesheets we are grabbing for a user to ensure they are properly prepared 
+        for the user - i.e. any missing days are added, etc. 
+    */
+
+    // Fetches timesheets and properly formats them to our frontend data versions. 
+    public static async fetch_user_timesheets(userid: string) {
+        //Grab timesheets from DB 
+        var timesheets =  await UserTimesheets(userid); 
+        //Convert to Frontend equivalents and convert 
+        timesheets = this.format(timesheets); 
+        
+
+        return DBToFrontend.convertTimesheets(timesheets); 
+    }
+
+    // Formats a list of backend / database timesheets to the frontend equivalents.   
+    public static format(timesheets: timesheetSchemas.TimeSheetSchema[]) : timesheetSchemas.TimeSheetSchema[] {
+        const updatedTimesheets =  timesheets.map((timesheet) => {
+            const [updatedTimesheet, modified] =  this.validate(timesheet); 
+            if (modified) {
+                //If this timesheet was modified we should re-upload it 
+                WriteEntryToTable(updatedTimesheet); 
+            }
+            return updatedTimesheet; 
+        })
+        return updatedTimesheets
+        
+    }
+
+    // Main method all other future methods delegate to / would return to when we are processing a timesheet to convert to frontend 
+    private static validate(timesheet): [timesheetSchemas.TimeSheetSchema, boolean] {
+        //When more functions are introduced here, create logic to determine whether any modified it to return 
+        return this.ensure_all_days(timesheet); 
+    }
+
+    private static ensure_all_days(timesheet:timesheetSchemas.TimeSheetSchema): [timesheetSchemas.TimeSheetSchema, boolean] {
```

Comment:
super small, but can we follow the same style conventions for naming (camel case rather than underscores)?